unblob/processing.py

```diff
@@ -422,34 +423,24 @@ def calculate_unknown_chunks(
     chunks: List[ValidChunk], file_size: int
 ) -> List[UnknownChunk]:
     """Calculate the empty gaps between chunks."""
-    if not chunks or file_size == 0:
+    # we do not want whole file unknown chunks (!?)
```

Comment:
sentence is unclear. Maybe something like "carving out an unknown chunk covering the entire file would lead to an infinite loop" ?

Reply:
I did not think about the infinite loop here, as carving would happen somewhere else, this condition should be there to prevent the infinite loop.

I have copied this comment, not verbatim, but documenting a former decision - I remember adding the `(!?)` as I do not fully agree with/understand the decision. The original comment was here:

https://github.com/onekey-sec/unblob/blob/main/unblob/processing.py#L324

```py
# we don't consider whole files as unknown chunks, but we still want to
# calculate entropy for whole files which produced no valid chunks
```

I have removed this comment, as it was not here before, and just induces confusion.